src/_guides/bump-sh-tutorials/mocking-with-microcks.md

```diff
@@ -0,0 +1,368 @@
+---
+title: Mocking APIs with Microcks
+authors: phil
+excerpt: Create powerful API mocks to speed up client development using Microcks.
+date: 2024-07-15
+---
+
+You've created beautiful documentation for your API with Bump.sh, powered by OpenAPI, what's next? If you've been following the [API design-first workflow](../api-basics/dev-guide-api-design-first.md) the next phase is gathering feedback on the proposed design before investing loads of time building it. 
+
+Sharing the documentation is a good start, but you can get even better feedback by giving stakeholders a mock server to interact with. A mock server simulates an API, allowing stakeholders to see if all the data they need is available, and give feedback on how easily their workflows can be solved based on the endpoints in the API. This can be thought of like a study group, something user researchers are used to doing for frontends, but is just as valuable for APIs.
+
+This used to be done with "prototypes" or manually made mock servers, but seeing as Bump.sh users have already got the API described entirely by OpenAPI already, setting up mock servers is a whole lot easier using OpenAPI-powered tools like Microcks to do the hard work for you. Using OpenAPI instead of code means changes can be made to the mock server far quicker than getting the API developments to change loads of code.
```

Comment:
```suggestion
This used to be done with "prototypes" or manually made mock servers, but seeing as Bump.sh users have already got the API described entirely by OpenAPI already, setting up mock servers is a whole lot easier using OpenAPI-powered tools like [Microcks](https://microcks.io/) to do the hard work for you. Using OpenAPI instead of code means changes can be made to the mock server far quicker than getting the API developments to change loads of code.
```